pkg/apis/core/validation/validation.go

```diff
@@ -3776,32 +3778,43 @@ const (
 	// a sysctl segment regex, concatenated with dots to form a sysctl name
 	SysctlSegmentFmt string = "[a-z0-9]([-_a-z0-9]*[a-z0-9])?"
 
-	// a sysctl name regex
-	SysctlFmt string = "(" + SysctlSegmentFmt + "\\.)*" + SysctlSegmentFmt
+	// a sysctl segment regex contains a slash, concatenated with dots to form a sysctl name
+	SysctlSegmentContainSlashFmt string = "[a-z0-9]([-_/a-z0-9]*[a-z0-9])?"
 
 	// the maximal length of a sysctl name
 	SysctlMaxLength int = 253
 )
 
-var sysctlRegexp = regexp.MustCompile("^" + SysctlFmt + "$")
```

Comment:
keep the MustCompile as a package var, this is expensive to do on every validation check

Reply:
I haven't considered this before, thank you for your reminder.